shell/app/common/components/pure-log-roller/log-roller.tsx

```diff
@@ -103,8 +102,8 @@ export class LogRoller extends React.Component<IProps, IState> {
     } = this.props;
     const { fullScreen } = this.state;
     let logContent = rolling ? (
-      <div>
-        Loading... <IconLoading spin />
+      <div className="flex">
+        Loading... <ErdaIcon className="ml-1" type="loading" color="currentColor" spin />
```

Comment:
CurrentColor should be the default props

Reply:
modified